### DISCUSSION TITLE
Welcome to Japanese-Writing-Evaluator Discussions!

### DISCUSSION THREAD

**Discussion by Vhulenda88** [COS301-SE-2022/Japanese-Writing-Evaluator#26] [General]
General chat

**Comment by Vhulenda88** [comment]
@Maryam-Al-Mahdi  I changed the  location of the images to `assets/images/imagename.png`, you can change your references to the pictures accordingly

**Comment by Siphesihle05** [comment]
🎨 - 🎨 - Improve structure / format of the code.
⚡️ - ⚡ - Improve performance.
🔥 - 🔥 - Remove code or files.
🐛 - 🐛 - Fix a bug.
🚑 - 🚑 - Critical hotfix.
✨ - ✨ - Introduce new features.
📝 - 📝 - Add or update documentation.
🚀 - 🚀 - Deploy stuff.
💄 - 💄 - Add or update the UI and style files.
🎉 - 🎉 - Begin a project.
✅ - ✅ - Add or update tests.
🔒 - 🔒 - Fix security issues.
🔖 - 🔖 - Release / Version tags.
🚨 - 🚨 - Fix compiler / linter warnings.
🚧 - 🚧 - Work in progress.
💚 - 💚 - Fix CI Build.
⬇️ - ⬇️ - Downgrade dependencies.
⬆️ - ⬆️ - Upgrade dependencies.
📌 - 📌 - Pin dependencies to specific versions.
👷 - 👷 - Add or update CI build system.
📈 - 📈 - Add or update analytics or track code.
♻️ - ♻️ - Refactor code.
➕ - ➕ - Add a dependency.
➖ - ➖ - Remove a dependency.
🔧 - 🔧 - Add or update configuration files.
🔨 - 🔨 - Add or update development scripts.
🌐 - 🌐 - Internationalization and localization.
✏️ - ✏️ - Fix typos.
💩 - 💩 - Write bad code that needs to be improved.
⏪ - ⏪ - Revert changes.
🔀 - 🔀 - Merge branches.
📦 - 📦 - Add or update compiled files or packages.
👽 - 👽 - Update code due to external API changes.
🚚 - 🚚 - Move or rename resources (e.g.: files, paths, routes).
📄 - 📄 - Add or update license.
💥 - 💥 - Introduce breaking changes.
🍱 - 🍱 - Add or update assets.
♿️ - ♿ - Improve accessibility.
💡 - 💡 - Add or update comments in source code.
🍻 - 🍻 - Write code drunkenly.
💬 - 💬 - Add or update text and literals.
🗃 - 🗃️ - Perform database related changes.
🔊 - 🔊 - Add or update logs.
🔇 - 🔇 - Remove logs.
👥 - 👥 - Add or update contributor(s).
🚸 - 🚸 - Improve user experience / usability.
🏗 - 🏗️ - Make architectural changes.
📱 - 📱 - Work on responsive design.
🤡 - 🤡 - Mock things.
🥚 - 🥚 - Add or update an easter egg.
🙈 - 🙈 - Add or update a .gitignore file.
📸 - 📸 - Add or update snapshots.
⚗ - ⚗️ - Perform experiments.
🔍 - 🔍 - Improve SEO.
🏷️ - 🏷️ - Add or update types.
🌱 - 🌱 - Add or update seed files.
🚩 - 🚩 - Add, update, or remove feature flags.
🥅 - 🥅 - Catch errors.
💫 - 💫 - Add or update animations and transitions.
🗑 - 🗑️ - Deprecate code that needs to be cleaned up.
🛂 - 🛂 - Work on code related to authorization, roles and permissions.
🩹 - 🩹 - Simple fix for a non-critical issue.
🧐 - 🧐 - Data exploration/inspection.
